backend/hitas/tests/factories/owner.py

```diff
@@ -45,6 +45,7 @@ class Meta:
     name = factory.Faker("name")
     identifier = faker.identifier()
     email = factory.Faker("email")
+    create_conditions_of_sale = True
```

Comment:
`create_conditions_of_sale` sounds like a method 😅 
I would have gone with maybe something such as `bypass_conditions_of_sale`

Reply:
I'll change the name, it might get confusing.